server/tournaments/index.ts

```diff
@@ -7,42 +22,38 @@ const MAX_REASON_LENGTH = 300;
 const MAX_CUSTOM_NAME_LENGTH = 100;
 const TOURBAN_DURATION = 14 * 24 * 60 * 60 * 1000;
 const ALLOW_ALTS = false;
+
 Punishments.roomPunishmentTypes.set('TOURBAN', 'banned from tournaments');
 
-/** @type {{[k: string]: Object}} */
-let TournamentGenerators = Object.create(null);
-/** @type {{roundrobin: string, elimination: string}} */
-const generatorFiles = {
-	'roundrobin': 'generator-round-robin',
-	'elimination': 'generator-elimination',
+const TournamentGenerators = {
```

Comment:
```suggestion
const TournamentGenerators = {
	__proto__: null,
```

I vaguely remember a `constructor` crash if you don't do this.